testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/OIDCScopeTest.java

```diff
@@ -68,12 +68,12 @@
  */
 public class OIDCScopeTest extends AbstractOIDCScopeTest {
 
-    private static String userId = KeycloakModelUtils.generateId();
+    private static String userId;
 
     @Override
     public void configureTestRealm(RealmRepresentation testRealm) {
         UserRepresentation user = UserBuilder.create()
-                .id(userId)
+                .id(KeycloakModelUtils.generateId())
```

Comment:
Does this line differ then in the class: ResourceOwnerPasswordCredentialsGrantTest.java that id is not deleted but rather generated?